oneandone/src/test/java/org/apache/jclouds/oneandone/rest/features/ServerApiLiveTest.java

```diff
@@ -0,0 +1,269 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.jclouds.oneandone.rest.features;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.jclouds.oneandone.rest.domain.FixedInstanceHardware;
+import org.apache.jclouds.oneandone.rest.domain.Hardware;
+import org.apache.jclouds.oneandone.rest.domain.HardwareFlavour;
+import org.apache.jclouds.oneandone.rest.domain.Hdd;
+import org.apache.jclouds.oneandone.rest.domain.Image;
+import org.apache.jclouds.oneandone.rest.domain.Server;
+import org.apache.jclouds.oneandone.rest.domain.Status;
+import org.apache.jclouds.oneandone.rest.domain.Types;
+import org.apache.jclouds.oneandone.rest.domain.Types.ServerAction;
+import org.apache.jclouds.oneandone.rest.domain.options.GenericQueryOptions;
+import org.apache.jclouds.oneandone.rest.internal.BaseOneAndOneLiveTest;
+import org.testng.Assert;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertFalse;
+import static org.testng.Assert.assertNotNull;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+@Test(groups = "live", testName = "ServerApiLiveTest")
+public class ServerApiLiveTest extends BaseOneAndOneLiveTest {
+
+    private List<Server> servers;
+    private Server currentServer;
+    private Server fixedInstanceServer;
+    private HardwareFlavour currentFlavour;
+    private Hdd currentHdd;
+    private Image currentImage;
+
+    private ServerApi serverApi() {
+
+        return api.serverApi();
+    }
+
+    @BeforeClass
+    public void setupTest() {
+        currentServer = createServer("jclouds test");
+    }
+
+    @AfterClass(alwaysRun = true)
+    public void teardownTest() throws InterruptedException {
+        //turn on currentServer in order to be able to delete
+        assertNodeAvailable(currentServer);
+        turnOnServer(currentServer.id());
+
+        if (fixedInstanceServer != null) {
+
+            //delete fixed instance server once ready 
+            assertNodeAvailable(fixedInstanceServer);
+            deleteServer(fixedInstanceServer.id());
+        }
+        if (currentServer != null) {
+            //delete currentserver once ready
+            assertNodeAvailable(currentServer);
+            deleteServer(currentServer.id());
+        }
+    }
+
+    @Test(dependsOnMethods = "testListHardwareFlavours")
+    public void testCreateFixedInstanceServer() {
+        fixedInstanceServer = serverApi().createFixedInstanceServer(Server.CreateFixedInstanceServer.create(
+                "java test fixed instance 2", "testing with jclouds", FixedInstanceHardware.create(currentFlavour.id()), "7C5FA1D21B98DE39D7516333AAB7DA54",
+                null, "Test123!", null, Boolean.TRUE, null, null, null, null));
```

Comment:
For constructors taking so many values, consider adding an auto-value builder to the model class.